components/monitoring/logging/base/configure-logging.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  collection:
+    logs:
+      fluentd: {}
+      type: fluentd
+---
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
```

Comment:
can we have a descriptive name?

Reply:
we can not change this name. this is fixed by the clusterlogging operator. https://docs.openshift.com/container-platform/4.12/logging/cluster-logging-deploying.html